Sources/CairoGraphics/context/CairoContext.swift

```diff
@@ -100,6 +100,26 @@ public final class CairoContext: GraphicsContext {
         }
     }
 
+    public func draw(polygon: Polygon<Double>) {
+        guard polygon.points.count > 0 else {
+            return
+        }
+
+        markImageAsUnflushed()
+        context.setSource(color: polygon.color.asDoubleTuple)
+
+        context.move(to: polygon.points.first!.asTuple)
+        for point in polygon.points.suffix(polygon.points.count - 1) {
+            context.line(to: point.asTuple)
```

Comment:
I think, we should close off the path, at least when filling is disabled (or, if this is intended, perhaps rename `Polygon` to something more general, e.g. `Path`). Same goes for the CoreGraphics implementation.

Reply:
Good idea. I did the change in the Polygon itself, so the logic was just in one place, and added some tests. I also added a check that it wasn't already closed, which was the case for the geospatial data I'm using this to render.